### PR TITLE
Added simple test tool to perform object upload, download, verify

### PR DESCRIPTION
**Pull request by grandsuri**
Added simple test tool to perform object upload, download, verify

### CODE REVIEW

**Comment by velomatt** [comment]
@grandsuri Please sign in to SonarCloud

**Comment by grandsuri** [comment]
> @grandsuri Please sign in to SonarCloud

Done

File: dss_client/test/test_data_integrity.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    parser.add_argument("--password", "-pw", type=str, default="minio123", required=False, action='store',
+                        help='Minio endpoint password (Default: minio123)')
+    parser.add_argument("--size", "-s", type=int, default=1, required=False, action='store',
+                        help='Size in MB (Default: 1)')
+    parser.add_argument("--user", "-u", type=str, default="minio", required=False, action='store',
+                        help='Minio endpoint admin (Default: minio)')
+    return parser
+
+
+def create_client_connection(endpoint, user='minio', pw='minio123'):
```
Comment (nsarras) [inline dss_client/test/test_data_integrity.py:31]: do we want the user/pass exposed here?

**Review by somnathr — APPROVED** [review]

**Comment by somnathr** [comment]
Looks good

**Pull request merged**